build/.vsts-PR-internal-check-healthcare-shared-components-azureBuild-pipeline.yml

```diff
@@ -0,0 +1,149 @@
+name:  pr$(System.PullRequest.PullRequestNumber)-$(Date:yyyyMMdd)$(Rev:-r)
+
+parameters:
+- name: BuildConfiguration
+  default: release
+
+variables:
+- name:  vmImage
+  value: windows-latest
+
+resources:
+  repositories:
+  - repository: self
+    type: git
+    ref: master
+
+jobs:
+
+- job: Phase_1
+  displayName: Phase 1
+  cancelTimeoutInMinutes: 1
+  pool:
+    vmImage: $(vmImage)
+
+  steps:
+  - checkout: self
```

Comment:
Let's not use NuGet unless we really need it. Instead can we rely only on `dotnet`? `dotnet build` should take care of restoring as well